src/presence_analyzer/templates/base.html

```diff
@@ -0,0 +1,30 @@
+<!doctype html>
+<html lang=en>
+<head>
+    <meta charset=utf-8>
+    {% block title %}
+        <title>Presence analyzer</title>
+    {% endblock %}
+    <meta name="keywords" content="" />
+    <meta name="description" content=""/>
+    <meta name="author" content="STX Next sp. z o.o."/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="{{ url_for('static', filename='css/normalize.css') }}" media="all" rel="stylesheet" type="text/css" />
+    <link href="{{ url_for('static', filename='css/style.css') }}" media="all" rel="stylesheet" type="text/css" />
+    <script src="{{ url_for('static', filename='js/jquery.min.js') }}"></script>
+    <script type="text/javascript" src="https://www.google.com/jsapi"></script>
+    <script type="text/javascript">
+        google.load("visualization", "1", {packages:["corechart", "timeline"], 'language': 'pl'});
+    </script>
+    <script type="text/javascript" src="{{ url_for('static', filename='js/scripts.js') }}"></script>
+    {% block head %}
+    {% endblock %}
+</head>
+<body>
+    <div id="main">
+        {% include "header.html" %}
+        {% block body %}
+        {% endblock %}
```

Comment:
I named it body, because it was in body block. But now it should be named `content`, right